### PR TITLE
[BEAM-3987] Use gradle and not maven in sdks/python/container/run_val…

### PR DESCRIPTION
**Pull request by alanmyrvold**
Use gradle and not maven in sdks/python/container/run_validatescontainer.sh

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by alanmyrvold** [comment]
Run Python Dataflow ValidatesContainer

**Comment by alanmyrvold** [comment]
+r @herohde

File: sdks/python/container/run_validatescontainer.sh [hunk 1]
```diff
@@ -117,7 +105,7 @@ python setup.py nosetests \
 
 # Delete the container locally and remotely
 docker rmi $CONTAINER:$TAG || echo "Failed to remove container"
-gcloud container images delete $CONTAINER:$TAG || echo "Failed to delete container"
+echo "y" | gcloud container images delete $CONTAINER:$TAG || echo "Failed to delete container"
```
Comment (lukecwik) [inline sdks/python/container/run_validatescontainer.sh]: `gcloud --quiet ...`?

See: https://cloud.google.com/sdk/docs/scripting-gcloud

Comment (alanmyrvold) [reply sdks/python/container/run_validatescontainer.sh]: Done.

**Pull request merged**